src/main/java/com/gmail/val59000mc/customitems/UhcItems.java

```diff
@@ -43,28 +57,59 @@ public static void giveLobbyItemsTo(Player player){
 		}
 	}
 
-	public static void openTeamMainInventory(Player player, UhcPlayer uhcPlayer){
-		List<ItemStack> items = new ArrayList<>();
+	public static void openTeamMainInventory(Player player, UhcPlayer uhcPlayer) {
+		List<ItemStack> items = new ArrayList<>(Collections.nCopies(45, new ItemStack(Material.AIR)));
+
+		int maxSlots = 5 * 9;
```

Comment:
We specify a literal max size here, then define `maxSlots`. This is silly, pls fix.